testing/conftest.py

```diff
@@ -24,7 +24,7 @@ def testdir(request, httpserver_base_url):
     if 'testdir' not in item.funcargnames:
         return
 
-    testdir = request.getfuncargvalue('testdir')
+    testdir = request.getfixturevalue('testdir')
```

Comment:
This was caught when running with DeprecationWarning enabled. @davehunt

Reply:
Thanks @BeyondEvil. This was introduced in pytest 3.0.0, which we already depend on, so this is a very safe change.

Reply:
I'm going to be a pain and ask you to split this (and a few other things) out into a separate commit or pull request. This shouldn't break anything for anyone, but if it does it makes it easier to identify if it's not combined with an unrelated change.